tf_quant_finance/black_scholes/brownian_bridge.py

```diff
@@ -0,0 +1,193 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Methods for brownian bridges.
+
+These can be used in Monte-Carlo simulation for payoff with continuous barrier.
+Indeed, the Monte-Carlo simulation is inherently discrete in time, and to
+improve convergence (w.r.t. the number of time steps) for payoff with continuous
+barrier, adjustment with brownian bridge can be made.
+
+## References
+
+[1] Emmanuel Gobet. Advanced Monte Carlo methods for barrier and related
+exotic options.
+https://papers.ssrn.com/sol3/papers.cfm?abstract_id=1265669
+"""
+
+import tensorflow.compat.v2 as tf
+
+
+def brownian_bridge_double(*,
+                           x_start,
+                           x_end,
+                           variance,
+                           upper_barrier,
+                           lower_barrier,
+                           n_cutoff=3,
+                           dtype=None,
+                           name=None):
+  """ Computes probabilities of not touching the barriers for a Brownian
```

Comment:
Could you please update the docstring as described [here](http://google.github.io/styleguide/pyguide.html#381-docstrings)? Something like
"""Computes probabilities of not touching the barriers for a Brownian Bridge.

The Brownian bridge starts at ... 
"""

There needs to be a blank line between the first line and the rest
Also, the docstring for `brownian_bridge_single` needs to be updated